src/spandsp.mk

```diff
@@ -0,0 +1,27 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := spandsp
+$(PKG)_WEBSITE  := https://www.soft-switch.org
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 0.0.6pre21
```

Comment:
That isn't the latest release, but you may be better off using snapshots if the release is so old.

Reply:
The changes between the latest stable release and snapshots are minimal, thus i would prefer the latest official version which is supported. Ok?

Reply:
Any feedback?

Reply:
@tonytheodore  Another ping for a feedback